### PR TITLE
Create CV / Markdown

### PR DESCRIPTION
**Pull request by ChrisGerBy**
This creates a CV using the recommendations of EPAM HR and written in Markdown Syntax.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  	<meta charset="utf-8">
+  	<title>Krystsina Gerko | Curriculum Vitae</title>
+  	<link rel="stylesheet" href="style.css">
+  	<link href="https://fonts.googleapis.com/css?family=Francois+One|Russo+One" rel="stylesheet">
+  	<link rel="shortcut icon" href="images/favicon.ico" type="image/png">
+  </head>
+  <body>
+  	<div id="wrapper">
+  		<header>
+  			<h1 id="curriculum-vitae">Krystsina Gerko</h1>
```
Comment (LenkouAndrei) [inline index.html]: why do you use id here? it is not necessary руку and you do not use it in other places

File: index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  			<h2>Junior Front-End Developer</h2>
+			<nav>
+				<ul>
+					<li><a href="#about-me">About me</a></li>
+					<li><a href="#skills">Skills</a></li>
+					<li><a href="#projects">Projects</a></li>
+					<li><a href="#education">Education</a></li>
+
```
Comment (LenkouAndrei) [inline index.html]: extra row

File: index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+				</ul>
+			</nav>
+		</header>
+
+		<main>
+			<div id="about-me">
+				<div class="contacts">
+				<h3>My contacts</h3>
```
Comment (LenkouAndrei) [inline index.html]: you need tab here

File: index.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+					<li><b>E-mail: </b><a href="mailto:k.gerko007@gmail.com">k.gerko007@gmail.com</a></li>
+					<li><b>Phone: </b> <a href="tel:+3754476659**">+375-44-766-59-**</a></li>
+					<li><b>Skype: </b> <a href="skype:christina_moroz">christina_moroz</a></li>
+				</ul>
+				</div>
+
+				<div class="text">
+					<hr>
+					<div class="text-content">
+						<p class="photo"><img src="images/photoKG.jpg" width="200" height="220" alt="Krystsina Gerko"></p>
```
Comment (LenkouAndrei) [inline index.html]: 1. tag "p" is good enough for text, but not for images, you may use "div" instead
2. also, you write img on the nest line

File: index.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+						<p>Hello! Thanks for visiting my page. My name is Krystsina and I am Junior Front-End Developer. I am just starting my career, so I am very exciting about it! On this page you can find my contacts and skills. Thanks for the interest!</p>
+					</div>
+					<hr>
+				</div>
+			</div>
+
+			<div id="skills">
+				<h3>Skills</h3>
+					<div class="languages left-side">
```
Comment (LenkouAndrei) [inline index.html]: you do not need tab here, div is not inside h3

File: index.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+						<h4>Programming languages</h4>		
+						<div class="flex-skills">
+							<img src="images/html_logo.png" width="120" alt="HTML5">
+							<img src="images/css.png" width="120" alt="HTML5">
```
Comment (LenkouAndrei) [inline index.html]: duplicate in alt

File: index.html [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+							<img src="images/english.png" width="120" alt="English">
+							<img src="images/german.png" width="120" alt="German">
+							<img src="images/spanish.png" width="120" alt="Spanish">
+						</div>
+					</div>
+			</div>
+
+			<div id="projects">
+				<h3>Projects</h3>
+					<p><strong>My current job and site project: </strong><a href="https://www.belfresh.by/" target="_blank">www.belfresh.by</a></p>
```
Comment (LenkouAndrei) [inline index.html]: too long line with tags. try your lines not to be larger, than vertical line in your code redactor

File: index.html [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+					<li><a href="#skills">Skills</a></li>
+					<li><a href="#projects">Projects</a></li>
+					<li><a href="#education">Education</a></li>
+
+				</ul>
+			</nav>
+		</header>
+
+		<main>
+			<div id="about-me">
```
Comment (LenkouAndrei) [inline index.html]: you can use section instead in such cases

File: index.html [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+					<ul>
+					    <li>Belarusian State University / Faculty of International Relationship</li>
+					     <li>University of Leipzig (Germany) / Scholarship programm for 2 semesters</li>
+					     <li>Online courses: Codeacademy, HTML Academy</li>
+					</ul>
+			</div>		
+		</main>
+
+		<footer>
+			<p>Krystina Gerko +375-44-766-59-**</p>
```
Comment (LenkouAndrei) [inline index.html]: it is better to use <a href="tel:555-555-5555">555-555-5555</a> for phone, to give possability to call, clicking text for user. PS your real phone nuber is not necessary

File: index.html [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+					<hr>
+					<div class="text-content">
+						<p class="photo"><img src="images/photoKG.jpg" width="200" height="220" alt="Krystsina Gerko"></p>
+						<p>Hello! Thanks for visiting my page. My name is Krystsina and I am Junior Front-End Developer. I am just starting my career, so I am very exciting about it! On this page you can find my contacts and skills. Thanks for the interest!</p>
+					</div>
+					<hr>
+				</div>
+			</div>
+
+			<div id="skills">
```
Comment (LenkouAndrei) [inline index.html]: you can use section in such places for semantic

File: index.html [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+<html lang="ru">
+  <head>
+  	<meta charset="utf-8">
+  	<title>Krystsina Gerko | Curriculum Vitae</title>
+  	<link rel="stylesheet" href="style.css">
+  	<link href="https://fonts.googleapis.com/css?family=Francois+One|Russo+One" rel="stylesheet">
+  	<link rel="shortcut icon" href="images/favicon.ico" type="image/png">
+  </head>
+  <body>
+  	<div id="wrapper">
```
Comment (LenkouAndrei) [inline index.html]: use classes instead. id only for communication with js also better to use wrapper inside of header, main and footer. what if you have to make only header for hole page width?

File: style.css [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+	margin-bottom: 15px;
+	text-transform: uppercase;
+}
+
+h4 {
+	margin-bottom: 10px;
+}
+
+#wrapper {
+	width: 100%;
```
Comment (LenkouAndrei) [inline style.css]: why do you set width 100% for block level element?

File: style.css [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+	margin: 25px 0px;
+}
+
+nav ul {
+	list-style-type: none;
+}
+
+nav li {
+	display: inline-block;
+	background-color: #000066;
```
Comment (LenkouAndrei) [inline style.css]: simplify to #006

File: style.css [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+	border-right: 1px solid white;
```
Comment (LenkouAndrei) [inline style.css]: use hex color instead #fff

File: style.css [hunk 15]
```diff
@@ -0,0 +1,146 @@
...
+	display: block;
+	color: white;
+	padding: 10px;
+}
+
+nav a:hover {
+	background-color: #00CCCC;
+}
+
+#about-me {
```
Comment (LenkouAndrei) [inline style.css]: you can use id for anchor, but for style better to use classes

Comment (LenkouAndrei) [reply style.css]: it'll help you to avoid conflict of priorities

File: index.html (hunk 1 above)
Comment (ChrisGerBy) [reply index.html]: I use it to define the style (mainly width and bg-color) of the content block as it is not the same as body.
`#wrapper {
	width: 100%;
	max-width: 1024px;
	margin: auto;
	background-color: white;
	padding: 15px 40px; 
	box-sizing: border-box;
`}``

File: style.css (hunk 15 above)
Comment (ChrisGerBy) [reply style.css]: It is also used  for navigation on the page. 
In the nav-block I set <a href="#about-me">...</a>.
And then I set this id to the corresponding section.

File: index.html (hunk 2 above)
Comment (ChrisGerBy) [reply index.html]: Resolved.

File: index.html (hunk 3 above)
Comment (ChrisGerBy) [reply index.html]: Resolved.

File: index.html (hunk 4 above)
Comment (ChrisGerBy) [reply index.html]: Resolved.

File: index.html (hunk 5 above)
Comment (ChrisGerBy) [reply index.html]: Resolved.

File: index.html (hunk 6 above)
Comment (ChrisGerBy) [reply index.html]: Resolved.

File: index.html (hunk 7 above)
Comment (ChrisGerBy) [reply index.html]: Resolved.

File: index.html (hunk 9 above)
Comment (ChrisGerBy) [reply index.html]: Added the link <a href="tel:+375445555555">...</a> in the footer phone number.

**Pull request merged**